src/components/Missions/Missions.js

```diff
@@ -14,28 +14,17 @@ const Missions = () => {
     }
   }, []);
 
+  const isError = () => {
+    if (missions.error === true) {
+      return (<p className='mission_error_message'>There was an error loading your missions, please try to reload the page</p>);
+    }
+    return (<MissionsTable />);
```

Comment:
- [OPTIONAL] In line with the above comment, if we made the `MissionsTable` component use props, this is where we would be supplying the prop. For example:

```JavaScript
    return (<MissionsTable missions={missions} />);
```

And keep our code DRY.